source/Grid/Grid.js

```diff
@@ -63,6 +64,12 @@ export default class Grid extends Component {
     noContentRenderer: PropTypes.func.isRequired,
 
     /**
+     * Optional callback invoked on mouse-wheel activity.
+     * (event: SyntheticEvent): void
+     */
+    onWheel: PropTypes.func,
```

Comment:
I'm very much on the fence about this. My reasons for leaning towards removing it are:

1. It might not _help_ performance; according to what Markus said it might actually make it _worse_.
1. It seems inconsistent (which I care about from an API point of view) to offer a single event-handler pass-through like this.